client/src/components/RefreshButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import refresh_btn from "../stories/assets/refresh_btn.svg";
+import styled from "styled-components/macro";
+
+const SecondaryButton = styled.button`
+  border: none;
+  background: transparent;
+`;
```

Comment:
Since PrimaryButton and SecondaryButton look the same, maybe you can create a const IconButton and change the Icon with PropTypes?